tests/fsharp/core/load-script/script.bat

```diff
@@ -44,4 +44,6 @@ echo Test 16=================================================
 "%FSC%" ProjectDriver.fsx --nologo
 ProjectDriver.exe
 del ProjectDriver.exe
+echo Test 17=================================================
```

Comment:
np @taylorwood , i'll rebase, dont care about my pr, you already update the `tests/fsharp/core/tests_core.fs` so it's ok